### PR TITLE
Prepare release 1.1-rc.0

### PR DESCRIPTION
**Pull request by pstibrany**
**What this PR does**: Preparation for Release 1.1.

Based on master 1ea1d19.

### CODE REVIEW

**Comment by pstibrany** [comment]
Hmm, but how do I set `release-1.1` branch as "target" branch now?

**Comment by codesome** [comment]
Thanks for the new PR!

**Comment by pstibrany** [comment]
> Thanks for the new PR!

Thanks for your help! I will update release doc to be more specific about the correct procedure.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+Please make sure to review renamed metrics, and update your dashboards and alerts accordingly.
 
 * [CHANGE] Added v1 API routes documented in #2327. #2372
   * Added `-http.alertmanager-http-prefix` flag which allows the configuration of the path where the Alertmanager API and UI can be reached. The default is set to `/alertmanager`.
   * Added `-http.prometheus-http-prefix` flag which allows the configuration of the path where the Prometheus API and UI can be reached. The default is set to `/prometheus`.
   * Updated the index hosted at the root prefix to point to the updated routes.
   * Legacy routes hardcoded with the `/api/prom` prefix now respect the `-http.prefix` flag.
-* [CHANGE] The metrics `cortex_distributor_ingester_appends_total` and `distributor_ingester_append_failures_total` now includes a `type` label to differentiate between `samples` and `metadata`. #2336
+* [CHANGE] The metrics `cortex_distributor_ingester_appends_total` and `distributor_ingester_append_failures_total` now include a `type` label to differentiate between `samples` and `metadata`. #2336
 * [CHANGE] The metrics for number of chunks and bytes flushed to the chunk store are renamed from: #2463
```
Comment (gouthamve) [inline CHANGELOG.md:15]: Can you add a note that the metrics counted here are _after_ deduplication while the metrics before were pre-deduplication?

File: CHANGELOG.md [hunk 2]
```diff
@@ -47,20 +52,20 @@
 * [ENHANCEMENT] Allow 1w (where w denotes week) and 1y (where y denotes year) when setting `-store.cache-lookups-older-than` and `-store.max-look-back-period`. #2454
 * [ENHANCEMENT] Optimize index queries for matchers using "a|b|c"-type regex. #2446 #2475
 * [ENHANCEMENT] Added per tenant metrics for queries and chunks and bytes read from chunk store: #2463
-* [ENHANCEMENT] Experimental WAL: New metrics `cortex_ingester_wal_logged_bytes_total` and `cortex_ingester_checkpoint_logged_bytes_total` added to track total bytes logged to disk for WAL and checkpoints. #2497
+* [ENHANCEMENT] WAL: New metrics `cortex_ingester_wal_logged_bytes_total` and `cortex_ingester_checkpoint_logged_bytes_total` added to track total bytes logged to disk for WAL and checkpoints. #2497
   * `cortex_chunk_store_fetched_chunks_total` and `cortex_chunk_store_fetched_chunk_bytes_total`
```
Comment (gouthamve) [inline CHANGELOG.md:56]: These metric additions need to move to line 54 (below change #2463). Might have been a rebase that messed up the ordering.

**Pull request merged**